src/Grid/Grid.php

```diff
@@ -1616,6 +1635,11 @@ public function setDefaultOrder($columnId, $order)
         return $this;
     }
 
+    public function getDefaultOrder(): ?string
```

Comment:
Should this be a string, or does int make more sense?

Reply:
Should be string (it's asc/desc/column id)